cmd/lotus/daemon.go

```diff
@@ -27,10 +27,6 @@ var DaemonCmd = &cli.Command{
 	Name:  "daemon",
 	Usage: "Start a lotus daemon process",
 	Flags: []cli.Flag{
-		&cli.StringFlag{
```

Comment:
Yeah, let's keep this flag. Its very useful because we don't have a way to initialize the repo without running the daemon first

Reply:
I've changed the repo exist to check datastore instead of config. This way we can copy the config in before repo init.

But I can restore the flag either way.

Reply:
Though i guess `echo '[API]\nListenAddress = "foo"' > $LOTUS_PATH/config.toml` isnt the worst thing